### PR TITLE
(DE-669) refactor(clickhouse): switch revenue aggregation from offerer to venue

### PR DESCRIPTION
**Pull request by valoumiaou**
# DE PR

## Describe your changes

Please include a summary of the changes:

This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 

Tag a reviewer if necessacy  @github/username 

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [ ] New table (non-breaking change)
- [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] My code passes CI/CD tests
- [ ] I updated README.md
- [ ] I have updated the dag
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I will create a review on slack and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### Added tests?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] ⏰ no, but I created a ticket

### CODE REVIEW

File: jobs/etl_jobs/internal/export_clickhouse/schema/intermediate/collective_booking.sql [hunk 1]
```diff
@@ -14,6 +15,6 @@ CREATE TABLE IF NOT EXISTS intermediate.collective_booking ON CLUSTER default
 )
 ENGINE = MergeTree
 PARTITION BY update_date
-ORDER BY (offerer_id, collective_booking_status, collective_offer_id)
+ORDER BY (venue_id, offerer_id, collective_booking_status, collective_offer_id)
```
Comment (cdelabre) [inline jobs/etl_jobs/internal/export_clickhouse/schema/intermediate/collective_booking.sql:18]: Pas sur que cela fonctionne avec les venue_id null ici ?

File: jobs/etl_jobs/internal/export_clickhouse/schema/tmp/intermediate_booking.sql [hunk 2]
```diff
@@ -1,13 +1,14 @@
 CREATE TABLE IF NOT EXISTS {{ dataset }}.{{ tmp_table_name }} ON cluster default
     ENGINE = MergeTree
     PARTITION BY update_date
-    ORDER BY (offerer_id, booking_status, offer_id)
+    ORDER BY (offerer_id, venue_id, booking_status, offer_id)
```
Comment (cdelabre) [inline jobs/etl_jobs/internal/export_clickhouse/schema/tmp/intermediate_booking.sql:4]: Same ici

**Review by cdelabre — COMMENTED** [review]
Questions

File: jobs/etl_jobs/internal/export_clickhouse/schema/intermediate/collective_booking.sql (hunk 1 above)
Comment (valoumiaou) [reply jobs/etl_jobs/internal/export_clickhouse/schema/intermediate/collective_booking.sql:18]: pas de venue_id null pour les offerer_id non-null

**Review by cdelabre — APPROVED** [review]

**Pull request merged**